java-api-core/src/com/kinvey/java/model/KinveyError.java

```diff
@@ -0,0 +1,37 @@
+package com.kinvey.java.model;
+
+import com.google.api.client.util.Key;
+
+public class KinveyError {
```

Comment:
The name of this class should be more specific, since it is for multi-insert errors.

Reply:
Renamed KinveyError class to KinveyBatchInsertError.